sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/PostgreSqlFlexibleServerDataEncryption.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Data encryption properties of a server. </summary>
+    public partial class PostgreSqlFlexibleServerDataEncryption
+    {
+        /// <summary> Initializes a new instance of PostgreSqlFlexibleServerDataEncryption. </summary>
+        public PostgreSqlFlexibleServerDataEncryption()
+        {
+        }
+
+        /// <summary> Initializes a new instance of PostgreSqlFlexibleServerDataEncryption. </summary>
+        /// <param name="primaryKeyUri"> URI for the key for data encryption for primary server. </param>
+        /// <param name="primaryUserAssignedIdentityId"> Resource Id for the User assigned identity to be used for data encryption for primary server. </param>
+        /// <param name="keyType"> Data encryption type to depict if it is System assigned vs Azure Key vault. </param>
+        internal PostgreSqlFlexibleServerDataEncryption(Uri primaryKeyUri, string primaryUserAssignedIdentityId, PostgreSqlFlexibleServerKeyType? keyType)
+        {
+            PrimaryKeyUri = primaryKeyUri;
+            PrimaryUserAssignedIdentityId = primaryUserAssignedIdentityId;
+            KeyType = keyType;
+        }
+
+        /// <summary> URI for the key for data encryption for primary server. </summary>
+        public Uri PrimaryKeyUri { get; set; }
+        /// <summary> Resource Id for the User assigned identity to be used for data encryption for primary server. </summary>
+        public string PrimaryUserAssignedIdentityId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
`PrimaryUserAssignedIdentityId` should be a resource identifier.

Reply:
Fixed.